pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AvgMVAggregationFunction.java

```diff
@@ -43,8 +45,24 @@ public AggregationFunctionType getType() {
   public void aggregate(int length, AggregationResultHolder aggregationResultHolder,
       Map<ExpressionContext, BlockValSet> blockValSetMap) {
     BlockValSet blockValSet = blockValSetMap.get(_expression);
-    double[][] valuesArray = blockValSet.getDoubleValuesMV();
 
+    if (blockValSet.getValueType() == DataType.BYTES) {
+      // Serialized AvgPair
+      byte[][] bytesValues = blockValSet.getBytesValuesSV();
```

Comment:
@Jackie-Jiang This would only be tested in Integration Tests, I don't see a away to test the code block with a unit test. I tried modifying SerializedBytesQueriesTest to support AVGMV on a multi-value field, but `blockValSet.getBytesValuesSV();` code block fails as we expect single value but its not single value (it would be single value only when data is read from startree index).